less/defaults.less

```diff
@@ -263,12 +264,21 @@ textarea {
 }
 
 input[type="text"],
+input[type="number"],
 input[type="search"],
 input[type="password"],
 textarea {
   border: 1px solid @light-gray;
   box-shadow: inset 1px 1px 4px 0 @light-gray;
   width: 100%;
+  &.error {
```

Comment:
I recently read trello's CSS guide and have been convinced of the `mod-` and `is-` prefixes.  What do you think about calling this `mod-has-error` or `is-invalid`?